app/src/pages/organization/organizationProjectsPage/organizationProjectsPage.jsx

```diff
@@ -72,34 +72,34 @@ export const OrganizationProjectsPage = () => {
     );
   };
 
+  const getEmptyPageState = () => {
+    organizationLoading ? (
+      <div className={cx('loader')}>
+        <BubblesLoader />
+      </div>
+    ) : (
+      <EmptyPageState
+        hasPermission={hasPermission}
+        label={label}
+        description={description}
+        icon={<PlusIcon />}
+        buttonTitle={buttonTitle}
+        emptyIcon={EmptyIcon}
+        onClick={showCreateProjectModal}
+      />
+    );
+  };
+
   return (
     <ScrollWrapper autoHeightMax={100}>
       <div className={cx('organization-projects-container')}>
-        {organizationLoading ? (
-          <div className={cx('loader')}>
-            <BubblesLoader />
-          </div>
-        ) : (
-          <>
-            <ProjectsPageHeader
-              hasPermission={hasPermission}
-              onCreateProject={showCreateProjectModal}
-            />
-            {isProjectsEmpty ? (
-              <EmptyPageState
-                hasPermission={hasPermission}
-                label={label}
-                description={description}
-                icon={<PlusIcon />}
-                buttonTitle={buttonTitle}
-                emptyIcon={EmptyIcon}
-                onClick={showCreateProjectModal}
-              />
-            ) : (
-              <ProjectsListTableWrapper projects={projects} />
-            )}
-          </>
-        )}
+        <>
+          <ProjectsPageHeader
+            hasPermission={hasPermission}
+            onCreateProject={showCreateProjectModal}
+          />
```

Comment:
In this case, isn't there a need for a loading state in the PropjectsPageHeader either?

Reply:
not needed